packages/html/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
-  "extends": "../../tsconfig-base.json",
+  "extends": "@frontside/tsconfig",
+  "compilerOptions": {
+    "esModuleInterop": true
+  },
```

Comment:
This is the default for `@frontside/tsconfig`

Reply:
I added extends from frontside config to the monorepo tsconfig and extend from it here, but for some reason it didn't work. I'll remove it, thanks